kas.yml

```diff
@@ -1,10 +1,10 @@
 header:
   version: 8
 
-distro: pce-xen
+distro: pce-base
 machine: pcengines-apu2
 target:
-  - xen-dom0-image
+  - core-image-minimal
```

Comment:
@Asiderr I wodner how it works, if you are building core-image-minimal, not the docker-image-minimal

Reply:
Maybe there should be multiple kas files.

Reply:
You could create kas-common.yml which contains most of the above file (except for the distro and target). Then, create kas-minimal, kas-docker, kas-xen files which include kas-common.yml and set the distro / target / any other settings as required. See: https://github.com/siemens/kas/blob/master/docs/userguide.rst#including-in-tree-configuration-files

Reply:
> I wodner how it works, if you are building core-image-minimal, not the docker-image-minimal

@macpijan I noticed that. I will fix it during splitting the kas file.